### PR TITLE
Phi-3 as relevance judge

### PR DESCRIPTION
**Pull request by elastic-thanos**
A notebook where Microsoft's phi-3 is used as a relevance judge

### CODE REVIEW

**Comment by qherreros** [comment]
I ran the full notebook locally, it works well! LGTM!

**Review by qherreros — APPROVED** [review]

File: supporting-blog-content/evaluating-search-relevance-part-2/phi3-as-relevance-judge.ipynb [hunk 1]
```diff
@@ -0,0 +1,790 @@
...
+   "metadata": {},
+   "source": [
+    "More specifically:\n",
+    "* `prompt_inputs`, specifies the list of attributes that need to be set in the prompt template. These attributes have the same name in the training data\n",
+    "* `prompt_template`, the prompt template to use\n",
+    "* `response_types`, the names of the expected output classes.\n",
+    "* `metadata`, the extra attributes that need to be preserved\n",
+    "* `max_output_tokens`, the maximum number of tokens that the LLM outputs"
+   ]
+  },
```
Comment (tveasey) [inline supporting-blog-content/evaluating-search-relevance-part-2/phi3-as-relevance-judge.ipynb:396]: Move to before?
```suggestion
```

File: supporting-blog-content/evaluating-search-relevance-part-2/phi3-as-relevance-judge.ipynb [hunk 2]
```diff
@@ -0,0 +1,790 @@
...
+  {
```
Comment (tveasey) [inline supporting-blog-content/evaluating-search-relevance-part-2/phi3-as-relevance-judge.ipynb:397]: merge with `POOL` definition

**Review by tveasey — APPROVED** [review]
Overall LGTM. Just a few very minor optional suggestions.

**Pull request merged**